src/components/Header/Header.tsx

```diff
@@ -23,7 +23,7 @@ export const Header: React.FC = () => {
               <NavLink
                 className={({ isActive }) =>
```

Comment:
Also, you have navLinks defined in the body od the Header component above in the code, it's better not to keep static variables inside the component body

move navLinks outside of the `export const Header: React.FC = () => {` or even to the separate file